cmd/release-controller-api/http_helper.go

```diff
@@ -257,17 +258,49 @@ func tableLink(config *releasecontroller.ReleaseConfig, tag imagev1.TagReference
 	return fmt.Sprintf(`<td class="text-monospace %s">%s</td>`, phaseAlert(tag), template.HTMLEscapeString(tag.Name))
 }
 
+func (c *Controller) GetReleasePayload(name string) *v1alpha1.ReleasePayload {
+	payload, err := c.releasePayloadLister.ReleasePayloads(c.releasePayloadNamespace).Get(name)
+	if err != nil {
+		klog.Warningf("Unable to locate releasepayload (%s): %v", name, err)
+		return nil
+	}
+	return payload
+}
+
+func generateVerificationStatusMap(payload *v1alpha1.ReleasePayload, status *releasecontroller.VerificationStatusMap) bool {
+	if status == nil || *status == nil {
```

Comment:
If `status == nil`, then `*status` will cause a panic. We should return early if `status == nil`